README.md

```diff
@@ -26,10 +26,47 @@ Requires:
 - ssh key added to github user account
 - setup a webhook pointing to the bors server using the `/github` endpoint
 
-1. Create a GitHub personal access token in the `bot` account. Add it to the bors config under `github/api-token`.
-2. Create a SSH key in the `bot` account.  Provide this key locally on the bot's machine. Add it to the bors config under `git/ssh-key-file`.
-3. Add a webhook for the repo that points to the bors server using the `/github` endpoint.  Configure it to use `application-json`, and provide the secret under `github/webhook-secret`
-
+1. Create a Github [application](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app) e.g. `bors-app`
+   1. Put `bors-<your_name>` as the application name
+   2. Put this repo's address as the homepage URL e.g. https://github.com/bors-rs/bors
+   3. Activate webhook, and put the URL to your server/github e.g. https://my-bors.domain/github
+   4. Repository Permissions
+      * Actions - Read and Write
+      * Checks - Read and Write
+      * Commit statuses - Read and Write
+      * Contents - Read and Write
+      * Deployments - Read-only
+      * Issues - Read and Write
+      * Metadata - Read-only
+      * Projects - Read and Write
+      * Pull Requests - Read and Write
+   5. Subscribe To Events
+      * Meta
+      * Check Run
+      * Commit comment
+      * Deployment Review
+      * Issue Comment
+      * Issues
+      * Label
+      * Project
+      * Project Card
+      * Project Column
+      * Pull Request
+      * Pull Request Review
+      * Pull Request Review Comment
+      * Pull Request Review Thread
+      * Push
+      * Repository
+      * Status
+      * Workflow Dispatch
+      * Workflow Job
+      * Workflow Run
+2. Create a Github [machine user](https://docs.github.com/en/developers/overview/managing-deploy-keys#machine-users) e.g. bors
+3. Create a GitHub personal access token in the `bot` account. Add it to the bors config under `github/api-token`.
+4. Create an SSH key in the `bot` account.  Provide this key locally on the bot's machine. Add it to the bors config under `git/ssh-key-file`.
+5. Add a webhook for the repo that points to the bors server using the `/github` endpoint.  Configure it to use `application-json`, and provide the secret under `github/webhook-secret`
+6. Add any CI and appropriate SSH Keys.  CircleCI requires an SSH key for a machine user for multiple repos (e.g. the bot above).  Then, it can be added as dependent steps in the config.
```

Comment:
is this always true? or only for (multiple) private repos?

Reply:
You only need an SSH key from a machine user for multiple repos.  You can use a specific one for a single repo otherwise.

I went with the machine user in the instructions because it is simpler.